lists.lisp

```diff
@@ -61,6 +61,17 @@ From Emacs Lisp."
   "Like `append1', but destructive."
   (nconc list (list item)))
 
+(defsubst prepend (&rest lists)
+  "Construct and return a list by concatenating LISTS in reverse order.
+
+    (prepend list-1 list-2)
+    ≡ (append list-2 list-1)"
+  (apply #'append (reverse lists)))
+
+(define-modify-macro prependf (&rest lists)
+  "Modify-macro for prepend. Prepends LISTS to the PLACE designated by the first argument."
+  prepend)
```

Comment:
You have the docstring and the symbol reversed here.

Reply:
Whoops! Right you are! I'll fix that now!

Reply:
@ruricolist Should be fixed now!